articles/relnotes/index.md

```diff
@@ -16,6 +16,19 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.xx.xxxx.xxxx
+
+*Release date: January 28th, 2020*
+
+This release contains the following:
+
+- New Microsoft.Quantum.SDK NuGet package, see the [README](https://github.com/microsoft/qsharp-compiler/blob/master/src/QuantumSdk/README.md) for more details
+- Added support for .NET Core 3.1, it is highly recommended to have version 3.1.100 install since building with older .NET Core SDK versions may cause issues
```

Comment:
```suggestion
- Added support for .NET Core 3.1, it is highly recommended to have version 3.1.100 installed since building with older .NET Core SDK versions may cause issues
```